### ISSUE TITLE
Create Technical Specification draft for Scribe

### ISSUE THREAD

**Issue by richpomfret** [FreeUKGen/New-Projects#4]
Initially brief/high-level technical description/plan for how we plan to deliver the project requirements. Should form basis of funding application and can become as detailed as needed (but not more than necessary).

Requires approval via TASc before we can complete.

**Comment by richpomfret** [comment]
Not applicable if we go ahead with the Zooniverse project.

**Comment by DeniseColbert** [comment]
To be reviewed to decide time requirements

**Comment by benwbrum** [comment]
Both Scribe and the Zooniverse Project Builder(see #43)  have changed substantially over the last two years.

The new version of Scribe (ScribeAPI) was released 2-3 years ago, but the organizations creating it experienced so much turnover in personnel and management priorities that it was very nearly orphaned with only us and the University of California-Davis seriously pursuing it.  Meanwhile the Zooniverse created Project Builder, allowing anyone to run an image-based crowdsourcing project on their servers.  The Project Builder had serious problems for our uses, however, as it did not support text entry of any sort.

Over the past year, both platforms have developed substantially.  Zooniverse has slowly added more and more text transcription functionality to the Project Builder, so that it may support our data entry needs and be about 80% of what we'd do if we built something from scratch.  Meanwhile, Scribe has been adopted and extended by the Yale Digital Humanities Lab and the Library of Congress Labs, giving it a much more robust code-base and user community.

Since the Zooniverse Project Builder is far lower cost of the two (requiring no programming at all, and being hosted by Zooniverse), I think the best way forward is to do a trail on that platform for a handful of pages from a project.  This could be started immediately, and need not be done by a developer (though the task configuration is a bit tricky).  If we find and enumerate the gaps between it and what we'd like, we should return to Scribe.
